### ISSUE TITLE
Solidify Marathi Firefox For Android Campaign

### ISSUE THREAD

**Issue by brianking** [mozilla/participation-org#123]
## Goal:

At the end of this heartbeat, we should have all the pieces in place (or a timeline for them) to enable a marketing campaign for the Marathi version of Firefox OS.
## Success:

The [marketing brief](https://docs.google.com/document/d/1SdBrG4rxVJcpKcYqyoqdIirQecx1QwjMWZjWYReYtMA/edit?usp=sharing) is fully filled out and a tentative date is set to launch the campaign.
## Related:
#99
## Roles:

Owner: @brianking
Involved: @chelseanovak
## Required:
- [x] Key data points are in place on audience for campaign (@brianking, [small], Due: 28 Aug)
- [x] Following up with PMM on messaging (@chelseanovak, [small], Due: 28 Aug)
- [x] 2-4 campaign tactics/channels are chosen by the community (@brianking, [small], Due: 1 Sep)
- [ ] Investigate feasibility of putting APK on SD cards (@brianking, [small], Due: 4 Sep)
- [ ] Determine creative needs (@chelseanovak, [medium], Due: 11 Sep)
- [x] Initial set of metrics (@brianking, [medium], Due: 11 Sep)
- [ ] Team members list (@brianking, [small], Due: 11 Sep)

**Comment by LucyeoH-zz** [comment]
**Review Buddy Notes:**
- In the first bullet did you mean "campaign" 
- for "2-4 campaign tactics/channels chosen by community" will this happen at a meeting? Via email? Wondering if there's an extra point missing from here like "create and circulate tactic options"
- for "investigate feasibility of putting APK on SD cards" wondering if this could be more specific "email x to investigate feasibility" or if it's a number of smaller tasks if it would be helpful to write them out? (Just a preference thing about how you like to work)
- What's the "team members list?"

Flags:
- You have a few things where you'll be waiting on external people (i.e. Chelsea), what's your plan for sharing/coordinating on timelines so this doesn't become a blocker? 

Otherwise this looks great! Let me know when there's more to review!

**Comment by chelseanovak** [comment]
Feedback from PMM will be in in the next 24 hours. Once the local team has considered which tactics they want to use I will work with them on building out a creative asset list. That won't block on anything.

**Comment by brianking** [comment]
Thank you Buddy @LucyeoH for the review. Answers:
- Yes, fixed.
- The campaign tactics will be chosen by the volunteer team, and then vetted by Chelsea and I. This is very much a locally driven campaign.
- Not sure what this means yet, but will change as it becomes clearer.
- The team members are the local volunteer team running the campaign, and the roles they will play.

As for the flag, we will coordinate via a weekly meeting. If you have any other suggestions let us know.

**Comment by brianking** [comment]
Meeting notes from Monday:

https://docs.google.com/document/d/1slzicOOLxX8d25yXjep9I5_xHftHCJAv0ryFlhyjB-I/edit#heading=h.9w54u3nqny3y

**Comment by brianking** [comment]
A quick note about data points. I haven't been able to track down specific data about Android numbers. In particular, the information we seek is:
- How many Android devices are in the region where Marathi is spoken?
- Of those, how many have access to the Google Play Store?
- How many use the Marathi locale version of Android (if it exists)?
- How many people speak Marathi and have Android phones?

Judging from my conversations so far this will be very difficult to get, if not impossible.

For Firefox, we are looking at just under 2k pings last month for mr-IN. We can check that again before the campaign kicks off.

**Comment by brianking** [comment]
Initial messaging can be found here:

https://play.google.com/store/apps/details?id=org.mozilla.firefox&hl=en
https://www.mozilla.org/en-US/firefox/android/

Local team to adapt.

**Comment by djaniketster** [comment]
Sorry for late reply.
Following marketing tactics can be used:
1. Talking about Firefox for Android (Marathi) in every local events.
2. Targeting school and colleges in and around the city.(Mostly the cities are Pune and Nashik)

This will be followed by extensive social media promotion.

The aim is to target the people who are from rural or/and semi-urban background. By telling the students about Firefox for Android (Marathi), we influence their parents to use Firefox.

Team members:
1. City based, since the marketing activity is going to be city based.
2. Marketing (Talking to School/College authorities).
3. Speakers.
4. Social Media Team(includes making creatives, content and strategy).

**Comment by ioana-chiorean** [comment]
Hey all, do you need any QA support for this? L10n check?

**Comment by brianking** [comment]
Thanks @djaniketster ... can you be more specific about number 1. What type of events are you talking about?

For number 2, are you sure they are the right audience? I think students would be more likely to speak English and use the English version. Remember, our goal is to get more usage of the Marathi locale version.

**Comment by djaniketster** [comment]
Thanks @brianking ... about number 1, There will be two types of events we are thinking of 
1. The regular events happening around the city like Maker Party, App Days etc.
2. The events we will specifically do for the promotion of Firefox for Android(Marathi) in various schools and colleges.

About number 2, the schools and colleges which we will be selecting for the events will have teaching medium in Marathi iitself and even if the teaching medium is in English, students are more comfortable in interacting in Marathi.

Moreover the amount of Marathi content which is generally circulated in social media and other IM services is HUGE.So, we need to make people aware about the browser in their language.

**Comment by chelseanovak** [comment]
Hey team, 
We need to work on determining the target audience for this campaign. This will help guide us with the way forward. Here are the details that we need from you:

Details we need: 
Age range:
How active they are on the web:
Brands they prefer:
Values that matter to them: 
What Firefox for Android in Marathi can offer them. 

So for example (I am completely making this up. We're looking to you all to describe the kind of person you think will want to use Firefox for Android in Marathi) 
This campaign is going to target a young professional type of user. 
age 22-28
active online 3+ hours a day
prefer brands that help them get work done, make their work easier, have some prestige 
these users are busy making their way in their new career

Let me know if you have questions on this as you work on it.

**Comment by brianking** [comment]
We've had some challenges on this one. One by one updates...
- Key data points are in place on audience for campaign:
  - We're not going to be able to get most of the data we want, such as the number of people who speak Marathi with Android devices. What we do have is number of users currently using Marathi Firefox for Android, so we can set a target based on that.
- Following up with PMM on messaging
  - We have the basic messaging. But we would like some local messaging created and tailored to the local market.
- 2-4 campaign tactics/channels are chosen by the community
  - We have some ideas, but are still deciding on them.
- Investigate feasibility of putting APK on SD cards
  - No progress. APKs in general are blocked by technical issues, but I haven't checked in on the status.
- Determine creative needs
  - TBD. Will be driven by tactics once chosen
- Initial set of metrics
  - Listed in the brief, but need to be refined.
- Team members list
  - Unknown

**Comment by brianking** [comment]
So most of these will in some shape or form need to be carried over to the next heartbeat. We should consider breaking them up into multiple issues, based on categories. e.g. marketing brief related, APK related, ...

**Comment by brianking** [comment]
Reopening because there were some issues not completed. See previous two comment.

**Comment by chandrakantd** [comment]
Hey Guys...Recently, I participated in "Vishwa Marathi Sahitya Sammelan" (World Marathi Literature Conference) at Port Blair, Andaman (5-6 Sept 2015). This was a great gathering of Marathi Loving people of all ages. It was good chance to leverage where I could have helped people Install Mozilla Firefox for Android on their devices. I have demonstrated Firefox for Desktop though. 

I get to interact with lot of such gatherings. If we can finalize on availability of APK on SD cards, it would be very helpful.

Second, we can arrange for small events on Saturdays (specifically) in and around Pune in various groups like Home Makers, Social groups who work on Languages, etc.
